include/boost/geometry/algorithms/detail/disjoint/areal_areal.hpp

```diff
@@ -39,15 +39,39 @@ namespace detail { namespace disjoint
 {
 
 
-template<typename Geometry>
+template <typename Geometry, typename Tag = typename tag<Geometry>::type>
+struct check_each_ring_for_within_call_covered_by
+{
+    template <typename Point, typename PiGStrategy>
```

Comment:
Hmm, PiGStrategy, funny name...
Maybe just Strategy here? A comment could explain that it should be a Point-in-Geometry strategy